CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Changes
 
 - Rename `navigation.processing` span to more expressive `Navigation dispatch to screen A mounted/navigation cancelled` ([#4423](https://github.com/getsentry/sentry-react-native/pull/4423))
+- Add RN SDK package to `sdk.packages` for Cocoa ([#4381](https://github.com/getsentry/sentry-react-native/pull/4381))
```

Comment:
@krystofwoldrich  We should also move the Android change to `Changes` too.

Reply:
The android change was already released in 6.5.0.